### PR TITLE
S4 resaux sociaux homepage

### PR DESCRIPTION
**Pull request by anjalaN**

### CODE REVIEW

File: public/assets/css/index.css [hunk 1]
```diff
@@ -99,8 +99,23 @@ header{
 	font-size: 1.1em;
 	text-align: center;
 } 
-
-
+.container-réseaux{
```
Comment (Esoriak) [inline public/assets/css/index.css]: .container-reseaux sans espace

File: public/assets/css/index.css [hunk 2]
```diff
@@ -99,8 +99,23 @@ header{
...
+	margin-left:10vw;
+	margin-top: 2vh;
+	margin-bottom: 15vh;
+	text-align: center;
+	height: 6vh;
+	display: flex;
+	flex-direction: row;
+	justify-content: center;	
+}
+.container-réseaux img{
```
Comment (Esoriak) [inline public/assets/css/index.css]: .container-reseaux img {

File: src/View/Home/index.html.twig [hunk 3]
```diff
@@ -21,16 +21,20 @@
...
 	<div class="container-blogs">
 		<div class="row-blogs">
-
 			{% for blog in blogs %}
 				{% include "Home/../components/_cardblog_display.html.twig" %}
 			{% endfor %}
 		</div>
 	</div>
-
+	<div class=container-réseaux>
```
Comment (Esoriak) [inline src/View/Home/index.html.twig:33]: <div class="container-reseaux">

**Review by Esoriak — CHANGES_REQUESTED** [review]
see the comment for changes needed

**Review by Esoriak — APPROVED** [review]

**Review by Chilluno — APPROVED** [review]

**Review by GuillaumeCregut — APPROVED** [review]

**Pull request merged**